theano/configdefaults.py

```diff
@@ -1238,21 +1252,20 @@ def default_blas_ldflags():
                                           "mk2_rt"]])
 
         # Anaconda
-        if "Anaconda" in sys.version or "Continuum" in sys.version:
-            # If the "mkl-service" conda package (available
-            # through Python package "mkl") is installed and
+        if "Continuum Analytics" in sys.version:
```

Comment:
Seems I am mistaken about `sys.version`, it can contain either `Anaconda` or `Continuum Analytics` depending on whether you installed `Anaconda` or `Miniconda`. Then there is also `conda-forge` to worry about. I am wondering whether to elide these checks altogether and just proceed to check the next parts instead: `import mkl` and `call_subprocess_Popen([mingw_w64_gcc, '-v'])`

What do you think? Otherwise the if statement looking for conda-y things will not be very pretty.